src/RuleCollection.php

```diff
@@ -743,7 +743,7 @@ public function changeRuleOrder($ID, $action, $condition = 0)
                 $result2 = $iterator2->current();
                 $other_ID = $result2['id'];
                 $new_rank = $result2['ranking'];
-                echo $current_rank . ' ' . $ID . '<br>';
+                echo $current_rank . ' ' . htmlescape($ID) . '<br>';
```

Comment:
ID is an integer, probably better to do:
```suggestion
                echo $current_rank . ' ' . (int) $ID . '<br>';
```